shared/shared_test.go

```diff
@@ -53,6 +53,12 @@ func TestFiatShamirReturnsCorrectNumOfIndices(t *testing.T) {
 	require.Len(t, FiatShamir(challenge, 20, 15), 15)
 }
 
+// This test used to hang.
+// See https://github.com/spacemeshos/poet/issues/173
```

Comment:
I think this can be removed.

Reply:
Why? The comment describes the purpose of this test (which is to test a known past issue).